psalm-baseline.xml

```diff
@@ -119,13 +117,21 @@
       <code>bindValues</code>
     </TooManyArguments>
   </file>
+  <file src="src/Element/MultiCheckbox.php">
+    <NoValue>
+      <code>$value</code>
+    </NoValue>
+  </file>
```

Comment:
The `NoValue` issue here and in `Select` are the result of better type inference for setting value options via `setAttributes`, which is deprecated and invalid according to the docblock type of `array<string, scalar>`